js_modules/dagster-ui/packages/ui-core/src/assets/AssetActionMenu.tsx

```diff
@@ -93,9 +93,9 @@ export const AssetActionMenu = memo((props: Props) => {
             <MenuLink
               text="Show in group"
               to={
-                repoAddress && definition?.groupName
-                  ? workspacePathFromAddress(repoAddress, `/asset-groups/${definition.groupName}`)
-                  : ''
+                definition
+                  ? globalAssetGraphPathForGroup(definition.groupName, definition.assetKey)
+                  : '/asset-groups/'
```

Comment:
Does this need the trailing slash?

Reply:
It looks like it doesn't! It does redirect to the trailing slash version, but I think it looks cleaner without - will change